### DISCUSSION TITLE
Error Meaning - Error populating tooltip basic statistics data

### DISCUSSION THREAD

**Discussion by Heidinger35** [DaxStudio/DaxStudio#1117] [Q&A]
The error shown in the image below populates on my log screen sporadically. It appears to occur when I hovers over certain fields in the Metadata pane but I can not seem to locate specific columns, measures, or even the action that is causing this error.

Insight on what this error means and what is causing is would be helpful.

![image](https://github.com/DaxStudio/DaxStudio/assets/56665555/d519314a-4277-479f-a958-42c80b2d70d5)

**Comment by dgosbell** [comment]
What version of DAX Studio are you running. This error is possibly related to the data type of the column.

The tooltips basic statistics are the following
![image](https://github.com/DaxStudio/DaxStudio/assets/8241830/2dfd838a-1fac-4d6c-b68c-ebf1572eaa2d)

And there was a fix in 3.0.7 for an issue where boolean columns were generating errors for tooltips.

If this is not your issue can you reproduce this warning then go into Help > About then click on the link for the Log Folder and check the most recent log file. It should have some diagnostic information that I can use to isolate this issue. I've created an issue here #1118 to track this problem so it would help if you could upload the log file to that (you cannot upload attachments to discussions like this one)

**Reply by Heidinger35** [reply]
I am using version 3.0.7.

Could this be related to the dataset I'm connected to being direct query? One of the error messages seems directly related to this.

Two different error messages are produced. They seem to vary by table and not by data type. All fields of the same table, regardless of data type, produce either message 1 or 2 and the associated tooltip behavior outlined in the image. Measure fields do not produce an error.

![image](https://github.com/DaxStudio/DaxStudio/assets/56665555/9d7bf4a4-dcd2-4796-8286-6908822ef502)

I will provide the log file if needed once I've had time to take a look at what it contains.

Thank you for the continued assistance here!
